converter/internal/staticconvert/internal/build/eventhandler.go

```diff
@@ -20,10 +20,16 @@ func (b *IntegrationsConfigBuilder) appendEventHandlerV2(config *eventhandler_v2
 	}
 
 	b.diags.AddAll(common.ValidateSupported(common.NotDeepEquals, config.SendTimeout, eventhandler_v2.DefaultConfig.SendTimeout, "eventhandler send_timeout", "this field is not configurable in flow mode"))
-	b.diags.AddAll(common.ValidateSupported(common.NotDeepEquals, config.CachePath, eventhandler_v2.DefaultConfig.CachePath, "eventhandler cache_path", "this field is not configurable in flow mode"))
 	b.diags.AddAll(common.ValidateSupported(common.NotDeepEquals, config.InformerResync, eventhandler_v2.DefaultConfig.InformerResync, "eventhandler informer_resync", "this field is not configurable in flow mode"))
 	b.diags.AddAll(common.ValidateSupported(common.NotDeepEquals, config.FlushInterval, eventhandler_v2.DefaultConfig.FlushInterval, "eventhandler flush_interval", "this field is not configurable in flow mode"))
 
+	if config.CachePath != eventhandler_v2.DefaultConfig.CachePath {
```

Comment:
The static mode's default location is different to the flow mode's default location. If users didn't define cache_path in static mode, they will get the flow mode's default which is different and may break their setup.

Reply:
Yes, but I consider this a separate issue - #6323